### DISCUSSION TITLE
Github Copilot Generating Valid OpenAI API Key

### DISCUSSION THREAD

**Discussion by somaniarushi** [community/community#124906] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I had an incident today of Copilot generating what is an OpenAI API Key that is fully functional. I had previously seen instances of this ~a year ago, and since then mitigating strategies had prevented this from happening at least to myself. I do not know how to provide metadata but a simple repro-case should be the following:
```
import openai
client = openai.Client()
def hit_openai(prompt):
    import openai
    openai.api_key = 
```
If this is insufficient, I am willing to provide access to my complete dev environment at the present moment as well.

Questions:
1. Is this expected behavior? Is this a regression that is currently being investigated? If not, is there planned work to ensure that the frequency of such incidents has been decreased?
2. Is there a way to alert the owner of that particular key? While I have no interest in using someone else's resources for my experiments, malicious actors could rack up severe charges.

Thanks!

**Comment by CopperEagle** [comment]
Just wanted to say thank you for flagging this and rather than take and run :heart:  I hope that GitHub staff sees this soon...

**Comment by davecheney** [comment]
Please contact support who will escalate to the appropriate development team.
